Cesium3DTilesSelection/src/Tileset.cpp

```diff
@@ -761,8 +776,22 @@ bool Tileset::_meetsSse(
     const double distance = distances[i];
 
     // Does this tile meet the screen-space error?
-    const double sse =
+    double sse =
         frustum.computeScreenSpaceError(tile.getGeometricError(), distance);
+    if (_options.enableDynamicScreenSpaceError) {
```

Comment:
this is the main code that applies dynamic SSE. The `calcDynamicSSEDensity()` isn't used currently. You can see the graph of the formula in this desmos https://www.desmos.com/calculator/qtajjjczqc

In the graph above, 
m -> camera height
c -> the minimum height that dynamic SSE will takes effect
f -> the maximum height to remove dynamic SSE

a -> the near distance to apply dynamic SSE
b -> the far distance where we aggressively choose the coarse tile